collect_app/build.gradle

```diff
@@ -228,6 +228,9 @@ configurations.all {
         cacheChangingModulesFor(0, 'seconds')
 
         force 'org.objenesis:objenesis:2.6'
+
+        // Newer Jackson versions cannot be ued until Collect minSDK >= 26 (https://github.com/FasterXML/jackson-databind#android)
+        force 'com.fasterxml.jackson.core:jackson-databind:2.13.5'
```

Comment:
If we had that comment I mentioned above do we need this too? To me, it would be enough not to touch that dependency.